.github/workflows/deploy-dev.yaml

```diff
@@ -0,0 +1,19 @@
+name: Deploy to Dev Environment
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: mbta/actions/npm-install
+        with:
+          node-version: 14
+      - uses: mbta/actions/deploy-lambda
+        with:
+          role-to-assume: ${{ secrets.AWS_DEV_ROLE }}
```

Comment:
We're only provisioning a single AWS IAM role for deploying this application, correct? If that's the case, there's no need for environment-specific variables.

Reply:
fixed it